Makefile

```diff
@@ -275,6 +275,9 @@ requirements: virtualenv .sdist-requirements
 	# new version of requests) which we cant resolve at this moment
 	$(VIRTUALENV_DIR)/bin/pip install "prance==0.6.1"
 
+	# Install st2common to register metrics drivers
+	(cd ${ROOT_DIR}/st2common; ${ROOT_DIR}/$(VIRTUALENV_DIR)/bin/python setup.py install)
```

Comment:
I'm ok with this.  I thought making the requirements already did this because the last time I tested this branch, I didn't have to explicitly run the setup.  Can you use `python setup.py develop` instead?

Reply:
@m4dcoder I'm getting all kinda nasty side effects from doing this. I've tried develop and install trying to work around it. Found some strange WebOb bug that manifests itself only when `st2common` is installed and was able to resolve it by bumping the version. Now I'm investigating these config issues. Long story short just ignore my commits until I get the tests passing.